internal/services/http.go

```diff
@@ -0,0 +1,32 @@
+package services
+
+import (
+	"fmt"
+	"net/http"
+)
+
+func sendMetric(addr string, storageType string, name string, metric any) error {
+	url := fmt.Sprintf("http://%s/update/%s/%s/%v", addr, storageType, name, metric)
+	req, err := http.NewRequest(http.MethodPost, url, nil)
+	if err != nil {
+		return err
+	}
+
+	client := &http.Client{}
+	resp, err := client.Do(req)
```

Comment:
Please create an issue to use `Context` here

Reply:
++